ftw/zipexport/browser/zipexportview.py

```diff
@@ -22,6 +25,11 @@ class NoExportableContent(Exception):
 
 class ZipSelectedExportView(BrowserView):
 
+    def __init__(self, context, request):
+        super(ZipSelectedExportView, self).__init__(context, request)
+        registry = getUtility(IRegistry)
+        self.settings = registry.forInterface(IZipExportSettings)
```

Comment:
I would move that to `zip_selected` (locals) where it is used:
- it's only used at one place: in `zip_selected`
- `zip_selected` is only called once
- in browser views, loading things in `__init__` should be avoided when possible, since it is called while traversing, not while publishing and has no security; although the registry should not be that protected it is good practice